src/test/scala/nl.knaw.dans.easy.solr4files/components/DDMSpec.scala

```diff
@@ -0,0 +1,94 @@
+/**
+ * Copyright (C) 2017 DANS - Data Archiving and Networked Services (info@dans.knaw.nl)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package nl.knaw.dans.easy.solr4files.components
+
+import java.io.FileInputStream
+import java.net.{ HttpURLConnection, URL }
+
+import org.scalatest.{ FlatSpec, Matchers }
+
+import scala.util.Try
+import scala.xml.XML
+
+class DDMSpec extends FlatSpec with Matchers {
+
+  "solrLiteral" should "return proper values" in {
+    assume(canConnectToEasySchemas)
```

Comment:
instead of assuming that we can connect to easy-schema, maybe we should fail if we can't connect. In that case something is terribly wrong with easy01 and a failed test will show up more easily than an ignored test

Reply:
Then you can't build when offline while traveling.